vscode.d.ts

```diff
@@ -0,0 +1,14724 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
this file looks generated - did you mean to check it in?

Reply:
Yes, it's meant to be committed.